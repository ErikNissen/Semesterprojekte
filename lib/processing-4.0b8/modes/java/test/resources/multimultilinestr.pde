version https://git-lfs.github.com/spec/v1
oid sha256:fb53d0cfe1ff46a3d2f5a85300bce66204fd34b2b1d30f26af4e1421f719af31
size 192
