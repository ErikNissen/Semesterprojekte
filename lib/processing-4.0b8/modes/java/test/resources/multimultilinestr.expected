version https://git-lfs.github.com/spec/v1
oid sha256:da0bab4f3bfade52241dc791667cf19a5c35c0b481977061e1e4e710f2b5bc2a
size 911
